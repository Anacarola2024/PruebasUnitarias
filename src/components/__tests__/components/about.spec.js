import { mount } from '@vue/test-utils';
import { describe, it, expect } from 'vitest';
import AboutView from '@/views/AboutView.vue';

describe('about.vue', () => {
    it('matches the snapshot', () => {
      const wrapper = mount(AboutView);
      expect(wrapper.html()).toMatchSnapshot();
    });
  });