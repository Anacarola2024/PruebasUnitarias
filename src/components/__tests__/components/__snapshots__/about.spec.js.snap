// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`about.vue > matches the snapshot 1`] = `
"<div class="about">
  <h1>This is an about page.</h1>
</div>"
`;
